concerned-coyotes/README.md

```diff
@@ -1,2 +1,56 @@
-# Concerned Coyotes
-This is the default README of your team's project. Please replace this by a README with more information on your project. At the very least, your README should contain information on how to set-up and run your project.
+#  Concerned Coyotes
+
+### Project overview
+
+Our project is an app for early risers. It's meant to provide information for users, who get up early which includes news, a to-do list for the day, a weather forecast and interesting wikipedia articles.
+
+In order to provide individual information, a login is required.
+
+###  Setup
+
+In order to setup you need to run
+```
+pip install -r requirements.txt
+```
+
+in a Python 3.8 environment.
+
+Switch to the project directory
+
+```
+cd earlyinternet
+```
+
+Setup the project itself
+
+```
+python manage.py migrate
+python manage.py setup_tasks
+python manage.py run_tasks
+```
+
+### Run the application
+
+To run the application use
+
+```
+python manage.py runserver
+```
+
+and check it out with your favorite browser.
+
+### Missing features
+
+Unfortunately we didn't have enough time to include all our planned features. The one's that didn't make it are
+
+
+* Breakfest recipes
+* calender integration
+
+### References
+
+The background image was taken from [unsplash](https://unsplash.com/photos/IKbpXrInCJY)
+
+Icons were provided by [tablericons](https://tablericons.com/) 
```

Comment:
While you do provide attribution, to be fully compliant with the MIT of the icons you should include a copy of the original MIT license with the assets:

> The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.